src/sql/workbench/contrib/notebook/browser/notebookEditor.ts

```diff
@@ -449,6 +450,7 @@ export class NotebookEditor extends EditorPane implements IFindNotebookControlle
 				this.setSelection(p);
 				this._updateFinderMatchState();
 				this._setCurrentFindMatch(p);
+				this._finder.focusFindInput();
```

Comment:
Does this still scroll the matches into view if the next one is offscreen?

Reply:
Just checked and it seems that hitting enter scroll to the next match, but actually clicking on the down arrow in the find widget does not do anything.